app/models/agents/data_output_agent.rb

```diff
@@ -28,6 +28,7 @@ class DataOutputAgent < Agent
           * `ns_media` - Add [yahoo media namespace](https://en.wikipedia.org/wiki/Media_RSS) in output xml
           * `ns_itunes` - Add [itunes compatible namespace](http://lists.apple.com/archives/syndication-dev/2005/Nov/msg00002.html) in output xml
           * `rss_content_type` - Content-Type for RSS output (default: `application/rss+xml`)
+          * `response_headers` - A JSON object with any custom response headers. (example: `{"Access-Control-Allow-Origin": "*"}`)
```

Comment:
I might say `hash` or `object` instead of JSON object since most users will be using the GUI editor?